front/lib/api/assistant/global_agents.ts

```diff
@@ -59,17 +58,14 @@ class HelperAssistantPrompt {
  * - Add a case in getGlobalAgent with associated function.
  */
 async function _getHelperGlobalAgent(
-  auth: Authenticator,
```

Comment:
Auth was introduced by PR to manage the model config, so we don't need to send user anymore!